### PR TITLE
Add BeeGames.eu

### PR DESCRIPTION
**Pull request by aaexa-bg**
## Type of change

- [✔️] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [✔️] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [✔️] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [✔️] I have created the manifest.json with the required values

## Further comments

Dear Labymod Team,

we would like to add our Czech server to your server list on your website laby.net. I made a new file at github. We will be happy if our project can be on your websites.

Thank you!

Best regards,
aaexa_12 / Owner of BeeGames.eu / aaexa@beegames.eu

### CODE REVIEW

**Comment by JakobBruening** [comment]
Please fix the invalid json (you can also use something like https://jsonformatter.curiousconcept.com) or close the PR.

**Comment by aaexa-bg** [comment]
> Please fix the invalid json (you can also use something like https://jsonformatter.curiousconcept.com) or close the PR.

Hello,

thanks for message. I already changed now :).

Best reagrds,
aaexa_12

**Comment by JakobBruening** [comment]
Hey!

As you can see, there are still problems with the images: https://github.com/LabyMod/server-media/runs/4200294415?check_suite_focus=true

Additionally, please compress your images: https://compresspng.com

BR, Jakob :)

**Comment by aaexa-bg** [comment]
Hello @JakobBruening,

now is everything okay?

Thank you a lot!

File: minecraft_servers/BeeGames.eu/manifest.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+   ],
+   "social":{
+      "web":"https://beegames.eu/",
+      "web_shop":"https://store.beegames.eu/",
+      "web_support":"https://wiki.beegames.eu/",
+      "tiktok":"BeeGames.eu",
+      "facebook":"BeeGames.eu",
+      "instagram":"beegames.eu",
+      "discord":"https://discord.com/invite/kZG4NQAMJE",
+      "teamspeak":"ts3.beegames.eu"
```
Comment (JakobBruening) [inline minecraft_servers/BeeGames.eu/manifest.json:20]: Please remove the / at the end of every url here

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Just three more things: 

1. Please resolve the comment(s).
2. Please rename the directory (and the server_name) from "BeeGames.eu" to just "beegames" or something in lowercase to match the directory name conventions. 
3. Please fix formatting, example: `"web":"https"` should be `"web": "https"`

File: minecraft_servers/BeeGames.eu/manifest.json (hunk 1 above)
Comment (aaexa-bg) [reply minecraft_servers/BeeGames.eu/manifest.json]: Hello @JakobBruening,

I can´t change a name of direcotry. But the other things I changed. Thanks.

**Comment by JakobBruening** [comment]
1. Please change the directory name to "beegames"
2. Please change the "server_name" in the manifest.json to "beegames" (instead of BeeGames)

**Comment by aaexa-bg** [comment]
> 1. Please change the directory name to "beegames"
> 2. Please change the "server_name" in the manifest.json to "beegames" (instead of BeeGames)


IM not able to change name of directory I don´t know how.

**Comment by aaexa-bg** [comment]
> > 1. Please change the directory name to "beegames"
> > 2. Please change the "server_name" in the manifest.json to "beegames" (instead of BeeGames)
> 
> IM not able to change name of directory I don´t know how.

I created a new folder @JakobBruening.

**Review by JakobBruening — APPROVED** [review]
Looks good now. Thank you

**Pull request merged**